tools/tesseratos/src/tesseratos/asset_explorer/plugin.cpp

```diff
@@ -14,10 +15,14 @@ using cubos::engine::AnyAsset;
 using cubos::engine::Assets;
 using cubos::engine::Cubos;
 using cubos::engine::Settings;
-using cubos::engine::Toolbox;
 
 using namespace tesseratos;
 
+CUBOS_REFLECT_IMPL(AssetExplorerTool)
+{
+    return cubos::core::ecs::TypeBuilder<AssetExplorerTool>("AssetExplorerTool").withField("isOpen", &AssetExplorerTool::isOpen).build();
```

Comment:
I don't think you need the suffix 'Tool' in those types, they're all tools since they are in Tesseratos anyway